lib/osf-components/addon/components/search-result-card/component.ts

```diff
@@ -6,26 +6,28 @@ import { tracked } from '@glimmer/tracking';
 import Intl from 'ember-intl/services/intl';
 
 import SearchResultModel from 'ember-osf-web/models/search-result';
-import UserModel from 'ember-osf-web/models/user';
+import PreprintProviderModel from 'ember-osf-web/models/preprint-provider';
 
 interface Args {
     result: SearchResultModel;
+    provider?: PreprintProviderModel;
 }
 
 export default class SearchResultCard extends Component<Args> {
     @service intl!: Intl;
     @service store!: Store;
 
     @tracked isOpenSecondaryMetadata = false;
-    @tracked osfUser?: UserModel;
 
     @action
     toggleSecondaryMetadata() {
         this.isOpenSecondaryMetadata = !this.isOpenSecondaryMetadata;
     }
 
     get cardTypeLabel() {
-        return this.intl.t(`osf-components.search-result-card.${this.args.result.resourceType}`);
+        const preprintWord = this.args.provider?.preprintWord;
+        return (preprintWord && this.args.result.resourceType === 'preprint') ? preprintWord :
+            this.intl.t(`osf-components.search-result-card.${this.args.result.resourceType}`);
```

Comment:
nit: would generally prefer an explicit map from resource type to complete translation key, so it's possible to find how each key is used (and which are unused)